### PR TITLE
Portfolio skeleton

### PR DESCRIPTION
**Pull request by Leboroz**
I created 2 sections:
- Nav bar
- Header
both responsive to mobile.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @Leboroz,

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: views/index.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <label for="check" id="menu-mobile" class="toggle-nav"
+        ><i class="fa fa-bars"></i
+      ></label>
+      <ul id="menu" class="nav-bar-container">
+        <li class="list-item"><a href="#Home">Hello</a></li>
+        <li class="list-item"><a href="#Portfolio">Portfolio</a></li>
+        <li class="list-item"><a href="#About">About</a></li>
+        <li class="list-item"><a href="#Contact">Contact</a></li>
+      </ul>
+    </nav>
```
Comment (mke2111) [inline views/index.html:27]: - [ ] Instead of using the `nav` tag, enclose this section in a `header` tag

File: views/index.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <title>Portfolio and mobile skeleton</title>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <img class="mobile-nav-bar" src="../mobile-nav-bar.png" alt="figures" />
```
Comment (mke2111) [inline views/index.html:16]: - [ ] For the logo, wrap it around `a` tags. Remember it should redirect to a given page

File: views/index.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      <input type="checkbox" id="check" />
+      <label for="check" id="menu-mobile" class="toggle-nav"
+        ><i class="fa fa-bars"></i
+      ></label>
+      <ul id="menu" class="nav-bar-container">
+        <li class="list-item"><a href="#Home">Hello</a></li>
+        <li class="list-item"><a href="#Portfolio">Portfolio</a></li>
+        <li class="list-item"><a href="#About">About</a></li>
+        <li class="list-item"><a href="#Contact">Contact</a></li>
+      </ul>
```
Comment (mke2111) [inline views/index.html:26]: - [ ] Enclose the hamburg icon within `nav` tags. The navigation will ensure the application of semantic HTML

File: views/index.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+      </div>
+      <div class="decoration-section">
+        <img
+          class="mobile-decoration"
+          src="../mobile_decoration.png"
+          alt="figures"
+        />
+        <img class="decoration" src="../Thur.png" alt="figures" />
+      </div>
+    </header>
```
Comment (mke2111) [inline views/index.html:47]: - [ ] Now for this section, lets enclose it within `section` tags

File: views/index.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      <ul id="menu" class="nav-bar-container">
+        <li class="list-item"><a href="#Home">Hello</a></li>
+        <li class="list-item"><a href="#Portfolio">Portfolio</a></li>
+        <li class="list-item"><a href="#About">About</a></li>
+        <li class="list-item"><a href="#Contact">Contact</a></li>
+      </ul>
+    </nav>
+    <header class="container">
+      <div class="header-section">
+        <div class="header">Leonardo Albornoz</div>
```
Comment (mke2111) [inline views/index.html:30]: - [ ] This being the main title, enclose it within `h1` tags

File: styles/styles.css [hunk 6]
```diff
@@ -0,0 +1,208 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,600;0,700;0,800;0,900;1,100&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap');
+
+body {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin: 0 auto;
+  width: 100vw;
+}
```
Comment (mke2111) [inline styles/styles.css:8]: - [ ] Your page has a horizontal scroll, this implies that a given container is wider than the others. Give all your containers an equal width.

**Review by jmontanholi — APPROVED** [review]
Hello Leo Albornoz Cubillán!

Congratulations!! You have done a great job on this Project. Your PR is ready to merge so you can continue being great!

✅ You implemented everything the other code reviewer asked
☑️ You already started the desktop version of the portfolio, it was not required, but good job!
✅ You already implemented the menu button functionality for the mobile version, great job!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**